batchflow/tests/notebooks/tfmodel_test.ipynb

```diff
@@ -0,0 +1,351 @@
+{
```

Comment:
<p>I had to add <code>softsum</code> combine operation due to strange behaviour of pytest: running this notebook manually works just fine, while evaluating it via <code>eval</code> from script gives CUDNN error on <code>tf.concat</code>. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/363/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>